### PR TITLE
automate staging build process

### PR DESCRIPTION
**Pull request by WanjikuMac**

### CODE REVIEW

File: .github/workflows/staging-deploy.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Staging Deploy
+
+on:
+  push:
+    branches:
+    - alpha_build_process
```
Comment (TraceyOnim) [inline .github/workflows/staging-deploy.yml:20]: Tested this, its working fine. You can change the branch to develop so that it can be merged in

Comment (WanjikuMac) [reply .github/workflows/staging-deploy.yml]: Done

File: .github/workflows/staging-deploy.yml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+env:
+  PROJECT_ID: ${{ secrets.GCE_PROJECT }}
+
+jobs:
+  setup-build-publish-deploy:
+    name: Setup, Build, Publish, and Deploy
+    runs-on: ubuntu-latest
+    environment:
+      name: staging
+      url: 'http://35.245.13.116:4000'
```
Comment (Japhetb) [inline .github/workflows/staging-deploy.yml:31]: Why have you use maha staging IP address: 35.245.13.116 and not haha staging IP address: 34.150.173.73?

Comment (WanjikuMac) [reply .github/workflows/staging-deploy.yml:31]: we will change that after we get a static IP for haha, it doesn't affect updating the instance

Comment (TraceyOnim) [reply .github/workflows/staging-deploy.yml:31]: Let it be changed before merging, we might forget about it

**Pull request merged**